clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/SCV.scala

```diff
@@ -52,10 +56,13 @@ case class SCV(
   submittedAssembly: Option[String],
   interpretationDescription: Option[String],
   interpretationLastEvaluated: Option[String],
-  interpretationComments: Array[String]
+  interpretationComments: Array[String],
+  vcvTraitSetId: Option[String],
```

Comment:
```suggestion
  traitSetId: Option[String],
```
To match the table/column name we ultimately want to have.